src/adapters/adapter.js

```diff
@@ -163,4 +181,13 @@ export default class Adapter extends Configurable {
                     {[this.name]: adapterUserId});
   }
 
+  promptUser = (data, cb) => {
+    this.bot.emitter.emit(AO.WHISPER_USER,this.name, data);
+    if (this.prompts[data.userId]) {
+      this.prompts[data.userId].push({data, cb});
```

Comment:
Do we want to spam a user against all channels? Seems a little intense, but I'm not sure what else we'd do (besides picking a last-active or something)